dev/NavigationView/NavigationView.cpp

```diff
@@ -1903,6 +1922,24 @@ winrt::NavigationViewItemBase NavigationView::GetContainerForClickedItem(winrt::
         container = listView.ContainerFromItem(itemData).try_as<winrt::NavigationViewItemBase>();
     }
 
+    // still no container, lats look in footer list
+    if (container.Content() != itemData)
+    {
+        auto footerListView = IsTopNavigationView() ? m_topFooterNavListView.get() : m_leftFooterNavListView.get();
```

Comment:
>footerListView [](start = 13, length = 14)

Whenever possible try to declare your locals as const.

Reply:
I'm not so good in c++. What exactly you suggest